### PR TITLE
Updated clij update site

### PR DESCRIPTION
**Pull request by haesleinhuepf**
Updated link to clij documentation,
Added clij2 update site to ease development and testing

### CODE REVIEW

**Comment by haesleinhuepf** [comment]
Thanks! :-)

File: sites.yml [hunk 1]
```diff
@@ -280,8 +280,15 @@ sites:
     id: "clij"
     url: "https://sites.imagej.net/clij/"
     description: >-
-      GPU-acceleration for ImageJ. See
-      [documentation](https://clij.github.io/clij-docs)
+      GPU-acceleration for ImageJ. See https://clij.github.io/
```
Comment (ctrueden) [inline sites.yml]: Is there some reason you aren't using a Markdown link here?

File: sites.yml [hunk 2]
```diff
@@ -280,8 +280,15 @@ sites:
...
+    maintainers:
+      - "[Robert Haase](https://imagej.net/User:Haesleinhuepf)"
+
+  - name: "clij2"
+    id: "clij2"
+    url: "https://sites.imagej.net/clij2/"
+    description: >-
+      Experimental update site / incubator for the future release of clij2. https://clij.github.io/clij-advanced-filters
```
Comment (ctrueden) [inline sites.yml]: Or here?

**Comment by haesleinhuepf** [comment]
Hey @ctrueden,

thanks for the comment. I just made the links clickable. The reason behind was that the links were not readable in the window where the people see the list of update sites:
image

Thanks for your support!

Cheers,
Robert

**Comment by ctrueden** [comment]
> the links were not readable in the window where the people see the list of update sites:

Ah, I see. Thanks for the explanation.

The fix is probably to put the description as a column of the table, rather than a tooltip. The tooltips are not as easy to discover anyway.

Alternately, if the tooltip rendered HTML and had working hyperlinks, I guess this would be OK, but I think making hyperlinks work with tooltips might be infeasible. And I like the description as its own column better anyway. @frauzufall What do you think?

**Pull request merged**